pkg/network/master/master.go

```diff
@@ -63,10 +66,9 @@ func Start(networkConfig osconfigapi.MasterNetworkConfig, networkClient networkc
 		TypeMeta:   metav1.TypeMeta{Kind: "ClusterNetwork"},
 		ObjectMeta: metav1.ObjectMeta{Name: networkapi.ClusterNetworkDefault},
 
-		Network:          networkConfig.ClusterNetworkCIDR,
-		HostSubnetLength: networkConfig.HostSubnetLength,
```

Comment:
for API compatibility, we still need to set ClusterNetworkCIDR and HostSubnetLength, at least in the case where there is only a single network, so that code that relies on being able to read those values from the default ClusterNetwork will still work.

Reply:
This comment still applies